### DISCUSSION TITLE
Recurring _next/webpack-hmr?page=/ 500 error

### DISCUSSION THREAD

**Discussion by codingenvy** [vercel/next.js#22108] [Help]
Hi all, when opening the most basic next js application (you may check on [https://devproxy.eleundmartin.de](https://devproxy.eleundmartin.de)) the index.js loads just fine, but I receive recurring "_next/webpack-hmr?page=/" type 500 errors every 5 seconds.
The application is running on a dedicated server - on local machine everything runs smoothly.
server.js:
```
// server.js
const { createServer } = require('http');
const { parse } = require('url');
const next = require('next');

const dev = process.env.NODE_ENV !== 'production';
const app = next({ dev });
const handle = app.getRequestHandler();

app.prepare().then(() => {
  createServer((req, res) => {
    // Be sure to pass `true` as the second argument to `url.parse`.
    // This tells it to parse the query portion of the URL.
    const parsedUrl = parse(req.url, true);
    const { pathname, query } = parsedUrl;

    if (pathname === '/a') {
      app.render(req, res, '/a', query);
    } else {
      handle(req, res, parsedUrl);
    }
  }).listen(5004, err => {
    if (err) throw err;
    console.log('> Ready on http://localhost:5004');
  });
});
```

I have an additional nginx directive: 
```
location /  {
	proxy_pass    http://localhost:5004;
	proxy_http_version 1.1;
	proxy_set_header Upgrade $http_upgrade;
	proxy_set_header Connection 'upgrade';
	proxy_set_header Host $host;
	proxy_cache_bypass $http_upgrade;
}
```
I am sure the mistake is on my side - any help would be great!

**Comment by mkamalkayani** [comment]
Have you found the reason for the error?

I am getting a similar error every few seconds on my proxy server logs.

<-- GET /_next/webpack-hmr 
--> GET /_next/webpack-hmr 404

**Reply by xiangxu-meng** [reply]
Have you found the reason for the error?
I am getting a similar error every few seconds on my proxy server logs.

**Reply by mkamalkayani** [reply]
@xiangxu-meng yes, the hmr uses websockets which was not getting proxied. I used asset prefix in the config.js file to serve the files form localhost.

**Reply by Amirlasalle** [reply]
I am having the same issue, I even change the next.config.mjs file. I'm using next and I wanna figure out how I can get rid of this error "GET /_next/webpack-hmr HTTP/1.1" 500 105092
